app/server/lib/Triggers.ts

```diff
@@ -34,6 +34,9 @@ promisifyAll(RedisClient.prototype);
 // and the corresponding actions get the full values
 const docSession = makeExceptionalDocSession('system');
 
+// Specify valid status codes for webhook response.
+const validWebhookResponseStatusCodes = [200, 201];
```

Comment:
FWIW, there exists more status codes for http successful requests:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status#successful_responses

(And maybe there are room for also checking 3xx status codes, IDK, but I wouldn't mind at least to support only 2xx status codes for now)

Reply:
Maybe use

```
if (response.statusCode >= 200 && response.statusCode <= 299) => Valid ?
```

Reply:
Fine for me :)